hardy4yooz/DongTai
# Add auto check sca image name (#411)

* fix incorrect param

* Update install.sh to dtctl

* Fix docker-compose down with file

* Fix check load sql first

* Feat: simplify dtctl usage

* Add export schema and docker-compose.yaml

* Feat: Add logs tail for service

* Fix:add logs usage

* Fix: uncomment pull new images

* Fix data dir error

* Fix:ignore current version sql

* Fix data backup function

* Add dbrestore|dbbackup options

* Fix: use netstat instead of lsof

* Add dtctl usage doc

* Add unittest for dtctl

* Fix github action syntax error

* Fix add work dir for bats

* Add unittest in github action

* Remove scarf.sh gateway

* Update upgrade option from -v to -t

* Fix upgrade with different monut type

* Fix upgrade with original mount type

* Fix upgrade sql filter

* Fix update unittest

* Update default image registry to aliyun

* Update get project name to docker inspect

* Update checkout volume type when running

* Add auto check sca image name
